Setup/UpgradeData.php

```diff
@@ -50,4 +111,66 @@ protected function version_2_1_7(
             ]
         );
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function version_2_3_2(
+        ModuleDataSetupInterface $setup
+    ) {
+        $eavSetup = $this->eavSetupFactory->create(['setup' => $setup]);
+        $eavSetup->removeAttribute(
+            \Magento\Customer\Model\Customer::ENTITY,
+            'stripe_id'
+        );
+        /** @var CustomerSetup $customerSetup */
+        $customerSetup = $this->customerSetupFactory->create(['setup' => $setup]);
+
+        $customerEntity = $customerSetup->getEavConfig()->getEntityType('customer');
+        $attributeSetId = $customerEntity->getDefaultAttributeSetId();
+
+        /** @var $attributeSet AttributeSet */
+        $attributeSet = $this->attributeSetFactory->create();
+        $attributeGroupId = $attributeSet->getDefaultGroupId($attributeSetId);
+
+        $customerSetup->addAttribute(Customer::ENTITY, 'stripe_id', [
+            'type' => 'varchar',
+            'label' => 'Stripe Id',
+            'input' => 'text',
+            'required' => false,
+            'visible' => true,
+            'user_defined' => true,
+            'sort_order' => 1000,
+            'position' => 1000,
+            'system' => 0,
+        ]);
+        //add attribute to attribute set
+        $attribute = $customerSetup->getEavConfig()->getAttribute(Customer::ENTITY, 'stripe_id')
```

Comment:
we can remove this variable and call save after addData with chain-call

Reply:
Thanks - done